dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheStoreService.java

```diff
@@ -83,13 +89,36 @@
      * @param props service properties
      */
     @Activate
-    @FFDCIgnore(CacheException.class)
+    @FFDCIgnore({ CacheException.class, URISyntaxException.class })
     protected void activate(ComponentContext context, Map<String, Object> props) {
-        configurationProperties = props;
-
+    	configurationProperties = props;
```

Comment:
If I'm reading this correctly, this only modification to this line is to switch from standard 4 space indent to 3, such that it no longer aligns with the rest of this method.  As long as there are other fixes being made to this file, this can be corrected as well

Reply:
This shows up as the proper 4 spaces for me locally so hopefully Eclipse auto-fixed it so it is fixed when I push the updates for the previous comments.

Reply:
I see what is happening, it looks like you switched some of the spaces to a tab, which prints as 3 spaces in the browser where I'm reviewing this.

Reply:
Yeah, I realized that later.  I switched it back to spaces so hopefully that resolves it.